Source/iOS/OIDExternalUserAgentIOS.m

```diff
@@ -88,6 +92,35 @@ - (BOOL)presentExternalUserAgentRequest:(id<OIDExternalUserAgentRequest>)request
   BOOL openedSafari = NO;
   NSURL *requestURL = [request externalUserAgentRequestURL];
 
+  // iOS 12 and later, use ASWebAuthenticationSession
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 120000
+  if (@available(iOS 12.0, *)) {
+    __weak OIDExternalUserAgentIOS *weakSelf = self;
+    NSString *redirectScheme = request.redirectScheme;
+    ASWebAuthenticationSession *authenticationVC =
+        [[ASWebAuthenticationSession alloc] initWithURL:requestURL
+                                      callbackURLScheme:redirectScheme
+                                       completionHandler:^(NSURL * _Nullable callbackURL,
+                                                           NSError * _Nullable error) {
+      __strong OIDExternalUserAgentIOS *strongSelf = weakSelf;
+      if (!strongSelf)
```

Comment:
I know this was from another PR that also used this format, but I was meaning to go in and fix it - I just didn't want to further hold up that PR over a style nit. Could we not propagate the style violation?